examples/each.js

```diff
@@ -0,0 +1,69 @@
+const StringHelper = require('../index')
+
+let helper = new StringHelper()
+let people = [
+  { name: 'pedro', sex: 'm', age: 30 },
+  { name: 'leticia', sex: 'f', age: 21 },
+  { name: 'pablo', sex: 'm', age: 20 }
+]
+let titles = ['ID', 'Name', 'Sex', 'Age']
+
+const rows = function (element, index) {
+  this.cat('<tr>').prefix('\t')
+    .cat('<td>', index, '</td>')
+    .cat('<td>', element.name, '</td>')
+    .cat('<td>', element.sex, '</td>')
+    .cat('<td>', element.age, '</td>').end()
+    .cat('</tr>')
+}
+
+const titleRow = function (element) {
+  this.prefix('\t').cat('<th>', element, '</th>').end()
```

Comment:
You are abusing of `prefix` and `end` because it will be called for each `th` inside the same `tr`, you might think about using them before and after the `each` statement.